ci/jenkins/jobs/macros.yaml

```diff
@@ -253,41 +253,54 @@
           export GO111MODULE=on
           export GOPATH=/var/lib/jenkins/go
           export GOROOT=/usr/local/go
-          export GOCACHE="${WORKSPACE}/../gocache"
```

Comment:
@lzhecheng I think it's becoming more urgent that we extract the scripts from these YAML files and move them to separate files, to improve maintainability

Reply:
You are right. I will work on this.